Samples/Excel-Get-Started-With-Dev-Kit/README.md

```diff
@@ -0,0 +1,103 @@
+# Get Started with the Office Add-ins Dev Kit in Word
+
+<img src="./assets/thumbnail.png" width="800"/>
+
+This is a basic Excel add-in that sets background color of selected range. It uses a button in the add-in's task pane to do this.
+
+Excel add-ins are integrations that use the [Excel JavaScript APIs](https://learn.microsoft.com/office/dev/add-ins/reference/overview/excel-add-ins-reference-overview) and [Office Platform capabilities](https://learn.microsoft.com/office/dev/add-ins/overview/office-add-ins) to connect web services to the workbook.
+
+## How to run this sample
+
+### Prerequisites
+
+- Download and install [Visual Studio Code](https://visualstudio.microsoft.com/downloads/).
+- Install the latest version of [Office Add-ins Development Kit](https://marketplace.visualstudio.com/items?itemName=msoffice.microsoft-office-add-in-debugger) into Visual Studio Code.
+- Node.js (the latest LTS version). Visit the [Node.js site](https://nodejs.org/) to download and install the right version for your operating system. To verify if you've already installed these tools, run the commands `node -v` and `npm -v` in your terminal.
+- Office connected to a Microsoft 365 subscription. You might qualify for a Microsoft 365 E5 developer subscription through the [Microsoft 365 Developer Program](https://developer.microsoft.com/microsoft-365/dev-program), see [FAQ](https://learn.microsoft.com/office/developer-program/microsoft-365-developer-program-faq#who-qualifies-for-a-microsoft-365-e5-developer-subscription-) for details. Alternatively, you can [sign up for a 1-month free trial](https://www.microsoft.com/microsoft-365/try?rtc=1) or [purchase a Microsoft 365 plan](https://www.microsoft.com/microsoft-365/buy/compare-all-microsoft-365-products).
+  
+### Run the add-in from the Office Add-ins Development Kit
+
+1. **Download the sample code**
+
+   Open the Office Add-ins Development Kit extension and view samples in the **Sample gallery**. Select the **Create** button in the top-right corner of the sample page.
+   
+1. **Open the Office Add-ins Development Kit**
+    
+    Select the <img src="./assets/Icon_Office_Add-ins_Development_Kit.png" width="30" alt="Office Add-ins Development Kit"/> icon in the **Activity Bar** to open the extension.
+
+1. **Preview Your Office Add-in (F5)**
+
+    Select **Preview Your Office Add-in(F5)** to launch the add-in and debug the code. In the drop down menu, select the option **Desktop (Edge Chromium)**.
+
+    <img src="./assets/devkit_preview.png" width="500" alt="Screenshot shows Preview your Office add-in in Office Add-ins Development Kit"/>
```

Comment:
Try to avoid using terms like "Screenshot" or "Image" in alt-text. It's redundant because the screen reader will already note the object is an image.
```suggestion
    <img src="./assets/devkit_preview.png" width="500" alt="The 'Preview your Office Add-in' option in the Office Add-ins Development Kit's task pane."/>
```